oximeter/collector/src/lib.rs

```diff
@@ -256,24 +323,32 @@ impl Oximeter {
             address: server.local_addr().to_string(),
             collector_id: agent.id,
         };
+
+        let nexus_pool = {
+            let nexus_resolver: BoxedResolver =
+                if let Some(address) = config.nexus_address {
+                    Box::new(SingleHostResolver::new(address))
+                } else {
+                    Box::new(DnsResolver::new(
+                        service::Name(ServiceName::Nexus.srv_name()),
+                        bootstrap_dns,
+                        DnsResolverConfig {
+                            hardcoded_ttl: Some(tokio::time::Duration::MAX),
```

Comment:
Same here, is this what we want?

Reply:
See: https://github.com/oxidecomputer/omicron/pull/6745#discussion_r1784897730